elytron/src/main/resources/org/wildfly/extension/elytron/LocalDescriptions.properties

```diff
@@ -792,7 +792,10 @@ elytron.token-realm.jwt.issuer=A list of strings representing the issuers suppor
 elytron.token-realm.jwt.audience=A list of strings representing the audiences supported by this configuration. During validation JWT tokens must have an 'aud' claim that contains one of the values defined here.
 elytron.token-realm.jwt.public-key=A public key in PEM Format. During validation, if a public key is provided, signature will be verified based on the key you provided here.
 elytron.token-realm.jwt.key-store=A key store from where the certificate with a public key should be loaded from.
+elytron.token-realm.jwt.key-map=A map of named public keys for token verification.
 elytron.token-realm.jwt.certificate=The name of the certificate with a public key to load from the key store.
+elytron.token-realm.jwt.client-ssl-context=The SSL context to be used for fetching jku keys using HTTPS.
+elytron.token-realm.jwt.host-name-verification-policy=A policy that defines how host names should be verified when using HTTPS. Allowed values: "ANY".
```

Comment:
`DEFAULT` is also allowed now, right?

Note that there is no need to specify the allowed values in the description as they are already listed in the attribute metadata: 

````
"host-name-verification-policy" => {
                            "type" => STRING,
                            "description" => "A policy that defines how host names should be verified when using HTTPS. Allowed values: 'ANY'.",
                            "expressions-allowed" => true,
                            "required" => false,
                            "nillable" => true,
                            "allowed" => ["ANY"]
                        }
````